### ISSUE TITLE
Request: Cassette Tape load support

### ISSUE THREAD

**Issue by thepulserifle** [MiSTer-devel/Atari800_MiSTer#6]
Feel free to close this if it's not within the scope of the project, but there are titles for the Atari 800 that were only released on cassette tape (All Star Baseball from 1979 is an example). Would there be a possibility of developing/adding it to this core?

**Comment by ArKay74** [comment]
Yeah, I would also love to have tape support (I think most other 8-bit computer cores do?)

It's so easy to store something (e.g. a program you have typed in from a magazine) to a tape file in an emulator. Not being able to load those files in MiSTer is sad.

**Comment by lencio** [comment]
I have many programs on tape that I wrote in my childhood, and it would be wonderful to be able to run them on the atari800 core.
There are no plans to implement such functionality?

**Comment by lroby74** [comment]
I would like to have this function too

**Comment by sorgelig** [comment]
It's not easy to add such thing when you never had a real Atari 800.
Besides adding tape support i believe there are PC programs able to convert from tape to disk.
